tutorials/power_distribution/BUILD

```diff
@@ -0,0 +1,16 @@
+load("//bazel:defs:bzl", "cc_firmware")
```

Comment:
Your import statement should be `//bazel:defs.bzl`. There should only be one `:`, and it's to separate the folder from the file.

Reply:
Actually though, you don't need to import `cc_firmware` because you are not creating a firmware binary--you are creating a C library.